### ISSUE TITLE
Debugger crashes when an hxcpp breakpoint is hit in a file that is not uniquely named.

### ISSUE THREAD

**Issue by EricBishton** [HaxeFoundation/intellij-haxe#792]
(Reported by @m0rkeulv )

If two files with the same name (in different paths) occur in a project and a breakpoint is hit in one of them, the debugger tries to find the proper file by traversing the class path.  If the implicit (std) class path is, in fact, empty, then an immutable `HaxeClasspath.EMPTY_CLASSPATH` is returned.  Down the line, `HaxelibClasspathUtils.getFullClasspath()` tries to add the rest of the classpath elements to the empty class path and that causes an exception.

Several possible fixes exist:
1) Remove `HaxeClasspath.EMPTY_CLASSPATH` and use `new HaxeClasspath()` in its place.  (There are no tests for `== EMPTY_CLASSPATH` in the current code base.)
2) Make an new superclass for HaxeClasspath that doesn't include the mutating methods and fix the syntax errors.
3) Make all mutating methods return a HaxeClasspath that may not be the same as the one passed in.  (This is also error prone. albeit more stream-like.)  A way to do this is to remove `add()` and `addAll()` methods and add a `concat()` method.

**Comment by EricBishton** [comment]
@m0rkeulv fixed this locally by changing `getImplicitClassPath` in `HaxelibClasspathUtils`
```haxe
      if (null == libraryPath) {
//      return HaxeClasspath.EMPTY_CLASSPATH;
        return  new HaxeClasspath();
      }
```

**Comment by EricBishton** [comment]
This is fixed and will be in the 1.1.0 release.
